news/2024/2024-03-03-4dmwc-2024-concludes.md

```diff
@@ -0,0 +1,171 @@
+
+> 4DM has a number of major differences from MWC. Even as a coach for the MWC team for 3 seasons now (2021, 2022, 2023), I always have to readjust myself to fit in the 4DM environment
+>
+> The first difference is that in MWC you don’t often see underdogs, all the big names are there and you know their strengths and weaknesses and it’s not difficult to predict their performance. But in 4DM, there is always guaranteed to be at least 2-3 underdogs who will surprise everyone, like Taiwan 2024 or China 2023.
+>
+> So in MWC, I can relax on a few picks, knowing they will struggle at some too. But in 4DM, I always make sure everyone must do great or decent at all picks. Another difference is that in 4DM, players may (not always) struggle more with nerves. I have to always make sure everyone is mentally ready from the beginning of the tournament and keep everyone’s spirit alive till the end because it will likely be the key to victory.
+
+*This is the first time we have Vietnam standing on top in a World Cup setting! How does it feel for you knowing Vietnam has accomplished the championship? And what does it mean for the team and the Vietnamese community?*
+
+> It was unbelievable personally. I built the Vietnamese osu!mania community in 2020 with some of my trusted comrades, and since then I’ve always wanted Vietnam to prove itself in the tournament scene. I sought victories myself around 2019-2020 but injuries interrupted me so I became a coach and helped the next generations to seek them.
+>
+> We’ve had smiles and tears, but this victory on 4DM24 will remain among our community's greatest achievements. It is the symbol of our long-lasting fight, paid with sweat, tears, and endless determination.
+>
+> I’m forever grateful that my 6-year effort was not in vain and that our community received this triumph harmoniously.
+
+*It’s noted that you have decided to stop competing in international competitions moving forward. Although you have much more important priorities, will you still stay in touch with the Vietnamese community moving forward?*
+
+> Absolutely! My head said leave, but my heart said stay! I’ve spent a portion of my life on this game and the community I built, there’s no way I’ll leave it to rot. If the future Vietnam MWC/4DM teams need me as a coach, I’m always welcome. Furthermore, we have numerous projects for the community that require technology and creativity. I’m quite confident in my web development and imagination skills, so as long as I still have gray matter in my neurons, I’ll always be there to help.
+
+### SiLiS / Raveille
+
+[Raveille](https://osu.ppy.sh/users/1388767) needs no introduction in the mania community, but his work as a music producer should be taken notice of as he has made the Round of 16 Tiebreaker song for this tournament: [GAME MAKE OVER](https://osu.ppy.sh/beatmapsets/2123356#mania/4462335)!
+
+---
+
+*So what was the process behind making GAME MAKE OVER for 4DM 2024?*
```

Comment:
since we cant put emphasis on the song name via italics maybe do it with quotation marks?
```suggestion
*So what was the process behind making "GAME MAKE OVER" for 4DM 2024?*
```

Reply:
in the owc post we just left is as is

Reply:
your guys' choice really then

Reply:
As is it is if that's the format that was followed before.